beeline/pom.xml

```diff
@@ -233,6 +233,53 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <descriptorRefs>
+                <descriptorRef>jar-with-dependencies</descriptorRef>
+              </descriptorRefs>
+              <finalName>jar-with-dependencies</finalName>
+              <transformers>
+                <transformer implementation="com.github.edwgiz.mavenShadePlugin.log4j2CacheTransformer.PluginsCacheFileTransformer" />
```

Comment:
I was running in to this issue here https://stackoverflow.com/questions/48033792/log4j2-error-statuslogger-unrecognized-conversion-specifier
Tried a bunch of things and this seemed to work.
This piece of code already exists in hive codebase here https://github.com/apache/hive/blob/master/itests/hive-jmh/pom.xml#L102

This happens because of an issue that exists in log4j2 https://issues.apache.org/jira/browse/LOG4J2-673
This was the suggested workaround as mentioned in this https://stackoverflow.com/questions/34945438/log4j2-configuration-not-found-when-running-standalone-application-built-by-shad